components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ConfigurableViewFacet.java

```diff
@@ -43,6 +43,8 @@ public void configure(final Router router) {
     this.router = router;
   }
 
+  // FIXME: Move 'online' to core repository property
```

Comment:
This property came up when dealing with the UI as well, and I'm not sure what the intent for it is. https://github.com/sonatype/nexus-oss/pull/1183#discussion_r28408713